photon-lib/src/main/scala/com/linkedin/photon/ml/model/Coefficients.scala

```diff
@@ -14,7 +14,7 @@
  */
 package com.linkedin.photon.ml.model
 
-import breeze.linalg.{DenseVector, SparseVector, Vector, norm}
+import breeze.linalg.{Vector, norm}
```

Comment:
We can probably delete this class, since we need `DataFrame` `Column` primitives. The coefficients and variances will be replaced by regular Spark `Vector` objects.